### PR TITLE
(tlg0085)_all_files_but_001

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated URNs-conflicted with catalog, edited cts_xml, added in page level links.

### CODE REVIEW

File: data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml [hunk 1]
```diff
@@ -28,16 +28,15 @@
...
+            <title xml:lang="lat">Aeschyli Tragoediae</title>
                  <editor>Arthur Sidgwick</editor>
             <imprint>
               <pubPlace>Oxford</pubPlace>
               <publisher>Clarendon Press</publisher>
               <date>1902</date>
             </imprint>
           </monogr>
-          <ref target="https://hdl.handle.net/2027/njp.32101064224940">HathiTrust</ref>
+          <ref target="https://hdl.handle.net/2027/njp.32101064224940?urlappend=%3Bseq=64%3Bownerid=27021597765885648-72">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml:39]: I would suggest https://babel.hathitrust.org/cgi/pt?id=njp.32101064224940&view=2up&seq=66

better page and no "owner id" URL append in it

File: data/tlg0085/tlg003/__cts__.xml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0085" xml:lang="grc" urn="urn:cts:greekLit:tlg0085.tlg003">
-  <ti:title xml:lang="lat">Prometheus Vinctus</ti:title>
   <ti:title xml:lang="eng">Prometheus Bound</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg003.opp-grc3" workUrn="urn:cts:greekLit:tlg0085.tlg003">
+  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg003.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0085.tlg003">
     <ti:label xml:lang="grc">Προμηθευς δεσμωτης</ti:label>
```
Comment (lcerrato) [inline data/tlg0085/tlg003/__cts__.xml:4]: Προμηθεὺς Δεσμώτης

File: data/tlg0085/tlg003/tlg0085.tlg003.1st1K-grc1.xml
Comment (lcerrato) [inline data/tlg0085/tlg003/tlg0085.tlg003.1st1K-grc1.xml:1]: Προμηθεὺς Δεσμώτης

File: data/tlg0085/tlg002/__cts__.xml [hunk 3]
```diff
@@ -1,8 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0085" xml:lang="grc" urn="urn:cts:greekLit:tlg0085.tlg002">
-  <ti:title xml:lang="lat">Persae</ti:title>
   <ti:title xml:lang="eng">Persians</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg002.opp-grc3" workUrn="urn:cts:greekLit:tlg0085.tlg002">
+  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0085.tlg002">
     <ti:label xml:lang="grc">Περσαι</ti:label>
```
Comment (lcerrato) [inline data/tlg0085/tlg002/__cts__.xml:4]: Πέρσαι

File: data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml
Comment (lcerrato) [inline data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml:1]: Πέρσαι

File: data/tlg0085/tlg007/__cts__.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0085" xml:lang="grc" urn="urn:cts:greekLit:tlg0085.tlg007">
   <ti:title xml:lang="eng">Eumenides</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg007.opp-grc3" workUrn="urn:cts:greekLit:tlg0085.tlg007">
+  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg007.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0085.tlg007">
     <ti:label xml:lang="grc">Ευμενιδες</ti:label>
```
Comment (lcerrato) [inline data/tlg0085/tlg007/__cts__.xml:4]: Εὐμενίδες

File: data/tlg0085/tlg007/tlg0085.tlg007.1st1K-grc1.xml [hunk 5]
```diff
@@ -4,7 +4,7 @@
   <teiHeader xml:lang="eng">    
     <fileDesc>
       <titleStmt>
-        <title xml:lang="grc">Eumenides</title>
+        <title xml:lang="grc">Ευμενιδες</title>
```
Comment (lcerrato) [inline data/tlg0085/tlg007/tlg0085.tlg007.1st1K-grc1.xml:7]: Εὐμενίδες

File: data/tlg0085/tlg007/tlg0085.tlg007.1st1K-grc1.xml [hunk 6]
```diff
@@ -28,16 +28,15 @@
...
+            <title xml:lang="lat">Aeschyli Tragoediae</title>
             <editor>Arthur Sidgwick</editor>
             <imprint>
               <pubPlace>Oxford</pubPlace>
               <publisher>Clarendon Press</publisher>
               <date>1902</date>
             </imprint>
           </monogr>
-          <ref target="https://hdl.handle.net/2027/njp.32101064224940">HathiTrust</ref>
+          <ref target="https://hdl.handle.net/2027/njp.32101064224940?urlappend=%3Bseq=299%3Bownerid=27021597765885648-307">HathiTrust</ref>
```
Comment (lcerrato) [inline data/tlg0085/tlg007/tlg0085.tlg007.1st1K-grc1.xml:39]: https://babel.hathitrust.org/cgi/pt?id=njp.32101064224940&view=2up&seq=301

**Review by lcerrato — CHANGES_REQUESTED** [review]
Some of the Greek is missing accentuation and a couple of the HT links seem to have extra info in them.

File: data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0085/tlg002/tlg0085.tlg002.1st1K-grc1.xml:39]: I'll change that!

File: data/tlg0085/tlg003/__cts__.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0085/tlg003/__cts__.xml:4]: That's funny I copied the title for the cts_.xml file from the Scaife title page. Go figure. I will fix all this. Thanks @lcerrato

**Comment by AlisonBabeu** [comment]
Hi @lcerrato I have made all of the requested changes.

**Review by lcerrato — COMMENTED** [review]
made some updates

**Review by lcerrato — APPROVED** [review]

**Pull request merged**